src/Components/OATPropertyEditor/Internal/PropertiesModelSummary.tsx

```diff
@@ -249,7 +249,7 @@ export const PropertiesModelSummary: React.FC<IPropertiesModelSummaryProps> = (
                 setModelPath(value.trim());
                 // Log event for path
                 sendEventTelemetry({
-                    name: TelemetryEvents.modalChangePath,
+                    name: TelemetryEvents.modelChangePath,
```

Comment:
did you check with Riley about the name and version fields too?

Reply:
I told him about it, but added them because it was an easy win.